### ISSUE TITLE
Moving by double-clicking does not work properly

### ISSUE THREAD

**Issue by preslyp** [jakezatecky/react-dual-listbox#285]
In the latest version of Chrome 125, moving by double-clicking does not work properly. Sometimes it comes after multiple clicks, right away.

https://github.com/jakezatecky/react-dual-listbox/assets/5307313/a06ca2a8-c309-4395-9302-a35823ba0a0f

**Comment by jakezatecky** [comment]
Indeed, it appears that Chrome is now deselecting an option on second click, causing issues with double clicks.

It appears this is [an issue that has surfaced recently](https://support.google.com/chrome/thread/277009331/double-click-no-more-work-on-chrome-and-edge?hl=en). I am unsure what the appropriate work around for this would be.
